actions/ci-sonarqube-ts/action.yml

```diff
@@ -71,28 +81,27 @@ runs:
       with:
         fetch-depth: ${{ inputs.checkout-repo-fetch-depth }}
 
-    - name: Download all workflow run artifacts
-      uses: actions/download-artifact@65a9edc5881444af0b9093a5e628f2fe47ea3b2e # v4.1.7
```

Comment:
This only supports downloading artifacts only using workflow run id, which is hard to determine when the test and lint steps are run in different workflows

Reply:
x-repos? This doesn't require access to other repos. The artifacts downloads will be from different workflows within the same repo.

Reply:
ahh got it. that's cool. nice!

is there anyway to test this action? maybe create a fake ts dir in `releng-test`?

Reply:
I tested this in [functions-toolkit](https://github.com/smartcontractkit/functions-toolkit/actions/runs/9764321753/job/26952499524) repo, looks good

Reply:
Should we also make similar changes to ts-sonarqube-go action as well?

Reply:
yeah let's do that too :)

Reply:
Made the changes for `ci-sonarqube-go` and tested in [ci-sonarqube](https://github.com/smartcontractkit/chainlink-data-streams/actions/runs/9778712507/job/26996194231) step

Reply:
cool nice!